modules/dataquery/ajax/queryEqual.php

```diff
@@ -16,7 +16,10 @@
 $category = $_REQUEST['category'];
 $fieldName = $_REQUEST['field'];
 $value = $_REQUEST['value'];
-$value = is_numeric($value) ? $value : "\"$value\"";
+
+if(!is_numeric($value) && $value !== "null") {
+	$value = "\"$value\"";
+}
```

Comment:
I haven't been able to search for the string "null"
Is it required? IsNull work just fine